python-sdk/examples/fetch_policies_demo.py

```
@@ -0,0 +1,185 @@
+    start_time = time.time()
+    prompt = langwatch.prompts.get(
+        prompt_name, fetch_policy=FetchPolicy.MATERIALIZED_ONLY
+    )
+    end_time = time.time()
+
+    print(f"   Time taken: {end_time - start_time:.3f} seconds")
+    print(f"   Model: {prompt.model}")
+    print("   ✅ Success - loaded from local (guaranteed no API call)")
+
+
+def demo_cache_ttl(work_dir: Path, prompt_name: str):
```

Ruff (0.14.8)

Unused function argument: `prompt_name`

(ARG001)